### PR TITLE
Add PHP Core Roundup #4

### PR DESCRIPTION
**Pull request by Ayesh**

### CODE REVIEW

File: source/_posts/2022-07-28-php-core-roundup-4.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+CI/CD platforms that use Docker images can use the PHP 8.2 docker images available with various base images. GitHub Actions can also make use of <code>[shivammathur/setup-php](https://github.com/shivammathur/setup-php)</code> action, which supports PHP 8.2 builds.
+
+
+## Recent RFCs, Merged PRs, and Commits
+
+Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net web site](github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
```
Comment (saundefined) [inline source/_posts/2022-07-28-php-core-roundup-4.md:52]:
```suggestion
Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net web site](https://github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
```

File: source/_posts/2022-07-28-php-core-roundup-4.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+There is also a new proposal to refresh the php.net web site home page by Mike van Riel, with a relevant [Twitter thread here](https://twitter.com/mvriel/status/1542898919142789123) and a WIP [PR in here](https://github.com/php/web-php/pull/602).
+
+## RFC Updates
+
+Following are the RFCs discussed, voted, and implemented since our last update. 
+
+- **Implemented: [Random Extension](https://wiki.php.net/rfc/rng_extension) and a [follow-up](https://wiki.php.net/rfc/random_extension_improvement) RFC by Go Kudo**
+Go Kudo first-proposed [Object scoped RNG](https://wiki.php.net/rfc/object_scope_prng), back in 2020, and proposed a series of RFCs to improve PHP’s Random Number Generator. The RFCs voted and implemented in PHP 8.2 now refactors PHP functions such as `random_int()`, `random_bytes()`, `rand()`, `mt_rand()` to a new extension called `random`. This is merely an internal refactor, and it is not possible to compile PHP without the extension. \
+ \
```
Comment (saundefined) [inline source/_posts/2022-07-28-php-core-roundup-4.md]:
```suggestion
Go Kudo first-proposed [Object scoped RNG](https://wiki.php.net/rfc/object_scope_prng), back in 2020, and proposed a series of RFCs to improve PHP’s Random Number Generator. The RFCs voted and implemented in PHP 8.2 now refactors PHP functions such as `random_int()`, `random_bytes()`, `rand()`, `mt_rand()` to a new extension called `random`. This is merely an internal refactor, and it is not possible to compile PHP without the extension. 
 
```

File: source/_posts/2022-07-28-php-core-roundup-4.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+There is also a new proposal to refresh the php.net web site home page by Mike van Riel, with a relevant [Twitter thread here](https://twitter.com/mvriel/status/1542898919142789123) and a WIP [PR in here](https://github.com/php/web-php/pull/602).
+
+## RFC Updates
+
+Following are the RFCs discussed, voted, and implemented since our last update. 
+
+- **Implemented: [Random Extension](https://wiki.php.net/rfc/rng_extension) and a [follow-up](https://wiki.php.net/rfc/random_extension_improvement) RFC by Go Kudo**
+
+   Go Kudo first-proposed [Object scoped RNG](https://wiki.php.net/rfc/object_scope_prng), back in 2020, and proposed a series of RFCs to improve PHP’s Random Number Generator. The RFCs voted and implemented in PHP 8.2 now refactors PHP functions such as `random_int()`, `random_bytes()`, `rand()`, `mt_rand()` to a new extension called `random`. This is merely an internal refactor, and it is not possible to compile PHP without the extension. \
```
Comment (saundefined) [inline source/_posts/2022-07-28-php-core-roundup-4.md:69]:
```suggestion
   Go Kudo first-proposed [Object scoped RNG](https://wiki.php.net/rfc/object_scope_prng), back in 2020, and proposed a series of RFCs to improve PHP’s Random Number Generator. The RFCs voted and implemented in PHP 8.2 now refactors PHP functions such as `random_int()`, `random_bytes()`, `rand()`, `mt_rand()` to a new extension called `random`. This is merely an internal refactor, and it is not possible to compile PHP without the extension.
```

File: source/_posts/2022-07-28-php-core-roundup-4.md [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+
+   There is a new PHP class called `\Random\Randomizer`, that can be instantiated with a class object that implements the `\Random\Engine` interface. The extension provides a few implementations built-in, such as `\Random\Engine\Mt19937`, `PcgOneseq128XslRr64`, and `Xoshiro256StarStar`. \
```
Comment (saundefined) [inline source/_posts/2022-07-28-php-core-roundup-4.md:71]:
```suggestion
   There is a new PHP class called `\Random\Randomizer`, that can be instantiated with a class object that implements the `\Random\Engine` interface. The extension provides a few implementations built-in, such as `\Random\Engine\Mt19937`, `PcgOneseq128XslRr64`, and `Xoshiro256StarStar`.
```

**Review by saundefined — COMMENTED** [review]
Minor nits :)

**Comment by Ayesh** [comment]
You were spot on @saundefined, thank you so much. I pushed with the marked changes made to the file. Thank you.

**Comment by Ayesh** [comment]
Force-pushed with the changes as suggested on the Google doc. Thank you.

**Pull request merged**